fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -346,15 +346,16 @@ export default Ember.Controller.extend({
     },
 
     editGlobalServiceConfig: function() {
-      var planParams = [];
+      var planParams = Ember.A();
       this.get('model.plan.parameters').forEach(function(param) {
-        if (param.get('id').indexOf('::') === -1) {
+        // only show parameters without :: in the name
+        if (param.get('id').indexOf('::') !== -1) {
```

Comment:
I'm guessing that this is showing global parameters, rather than role parameters?  Role parameters are prefixed by the role name followed by '::' - e.g., 'Controller-1::foo'.